### PR TITLE
Update attachment file size limit

### PR DESCRIPTION
**Pull request by mafeld**

### CODE REVIEW

**Review by peiying2 — APPROVED** [review]

**Comment by flodolo** [comment]
Has anyone looked into other locales, and if this can be fixed without going through the agency?

**Comment by flodolo** [comment]
> Has anyone looked into other locales, and if this can be fixed without going through the agency?

Pushed a commit with the other locales.

@mafeld 
Do we need an update to the date as well?

**Comment by mafeld** [comment]
good point. I'll change the date

**Review by flodolo — APPROVED** [review]
Updated dates and versions per conversation with Michael, merging.

I guess we need to wait next week to push to prod.

File: it/firefox_relay_tos.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿# Termini di servizio di Firefox Relay
 
-Versione 1.3, in vigore dal 10 marzo 2022
-{: datetime="2022-03-10" }
+Versione 1.4, in vigore dal 10 settembre 2022
```
Comment (flodolo) [inline it/firefox_relay_tos.md:3]: And obviously I missed the day in the one language I speak. WIll fix in a separate commit.

**Pull request merged**